src/sql/workbench/contrib/notebook/test/electron-browser/notebookModel.test.ts

```diff
@@ -480,6 +480,24 @@ suite('notebook model', function (): void {
 
 	});
 
+	test('Should not delete custom metadata', async function (): Promise<void> {
```

Comment:
This is part of the other PR isn't it? Please revert any changes not related to your PR (you can still have them locally for testing purposes, just don't commit them)

Reply:
This was merge conflict. The changes are already in master